### PR TITLE
feat(docs): add versions story

### PR DESCRIPTION
**Pull request by gfellerph**
Added a story showing all interesting dependency version numbers used for this particular build of the documentation.

### CODE REVIEW

**Comment by swisspost-devs** [comment]
Preview environments are getting posted here, as soon as they are ready!

**Review by oliverschuerch — COMMENTED** [review]
Why diden't you use/extend our documentation/src/utils/version.ts file to get the versions of the required packages?

File: packages/documentation/src/stories/misc/versions.components.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import internetHeaderVersions from '@swisspost/internet-header/package.json';
+import componentsReactVersions from '@swisspost/design-system-components-react/package.json';
+import documentationVersions from '../../../package.json';
+
+const dependencies = [
+  stylesVersions,
+  componentsVersions,
+  internetHeaderVersions,
+  componentsReactVersions,
+];
```
Comment (oliverschuerch) [inline packages/documentation/src/stories/misc/versions.components.tsx:12]: Where is the components-angular (intranet-header) project?

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

File: packages/documentation/src/stories/misc/versions.components.tsx (hunk 1 above)
Comment (gfellerph) [reply packages/documentation/src/stories/misc/versions.components.tsx:12]: It's not a dependency of the documentation as it's not possible to document Angular components (yet).

**Review by alizedebray — APPROVED** [review]

File: packages/components/src/components/post-icon/post-icon.tsx [hunk 2]
```diff
@@ -184,7 +185,7 @@ export class PostIcon {
 
   render() {
     return (
-      <Host>
+      <Host data-version={version}>
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/components/src/components/post-icon/post-icon.tsx:188]: Top level would be enough, it's just very handy for debugging when someone sends you a link to some page you don't have to ask which version of X they are using.

**Pull request merged**